From f697f4fc41074bc3ad344ac41d47dcd8c4d7ac8a Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Mon, 9 May 2022 06:05:49 -0400
Subject: manifest remove win10/11 manifest support

Because some old Intel GPU drivers fail to support openGL when enabled
---
 resources/msw/manifests/x64.manifest                | 4 +---
 resources/msw/manifests/x64_dpi_aware_pmv2.manifest | 4 +---
 resources/msw/manifests/x86.manifest                | 4 +---
 resources/msw/manifests/x86_dpi_aware_pmv2.manifest | 4 +---
 4 files changed, 4 insertions(+), 12 deletions(-)

diff --git a/resources/msw/manifests/x64.manifest b/resources/msw/manifests/x64.manifest
index a5f27f64b9..c9957f49a6 100644
--- a/resources/msw/manifests/x64.manifest
+++ b/resources/msw/manifests/x64.manifest
@@ -34,8 +34,6 @@
       <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}" />
       <!-- Windows 8.1 -->
       <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}" />
-      <!-- Windows 10 -->
-      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
     </application>
   </compatibility>
-</assembly>
\ No newline at end of file
+</assembly>
diff --git a/resources/msw/manifests/x64_dpi_aware_pmv2.manifest b/resources/msw/manifests/x64_dpi_aware_pmv2.manifest
index 34e0ffc8e9..63d734e7a7 100644
--- a/resources/msw/manifests/x64_dpi_aware_pmv2.manifest
+++ b/resources/msw/manifests/x64_dpi_aware_pmv2.manifest
@@ -30,8 +30,6 @@
       <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}" />
       <!-- Windows 8.1 -->
       <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}" />
-      <!-- Windows 10 -->
-      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
     </application>
   </compatibility>
-</assembly>
\ No newline at end of file
+</assembly>
diff --git a/resources/msw/manifests/x86.manifest b/resources/msw/manifests/x86.manifest
index 2c8b801f87..90a0af465f 100644
--- a/resources/msw/manifests/x86.manifest
+++ b/resources/msw/manifests/x86.manifest
@@ -34,8 +34,6 @@
       <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}" />
       <!-- Windows 8.1 -->
       <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}" />
-      <!-- Windows 10 -->
-      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
     </application>
   </compatibility>
-</assembly>
\ No newline at end of file
+</assembly>
diff --git a/resources/msw/manifests/x86_dpi_aware_pmv2.manifest b/resources/msw/manifests/x86_dpi_aware_pmv2.manifest
index 8c2d961862..d59e5e0b2a 100644
--- a/resources/msw/manifests/x86_dpi_aware_pmv2.manifest
+++ b/resources/msw/manifests/x86_dpi_aware_pmv2.manifest
@@ -30,8 +30,6 @@
       <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}" />
       <!-- Windows 8.1 -->
       <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}" />
-      <!-- Windows 10 -->
-      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
     </application>
   </compatibility>
-</assembly>
\ No newline at end of file
+</assembly>
-- 
2.36.0.windows.1

